### ISSUE TITLE
Animating the placeholder text on homepage is bad for accessibility

### ISSUE THREAD

**Issue by dhruvkb** [WordPress/openverse-frontend#652]
## Description

Animating the placeholder text of the search bar on the homepage every 100ms is a big problem for accessibility. We should find alternative approaches for this animation.

## Reproduction

1. Visit homepage using AT.
4. Experience the horror.

## Expectation

The animation should be a purely ornamental aspect of the homepage, and should completely cease to exist for assistive technologies.

## Additional context

This issue was branched from the homepage PR #593 with a possible solution in https://github.com/WordPress/openverse-frontend/pull/593#issuecomment-1017668491.

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

**Comment by zackkrida** [comment]
We can probably fix this by putting the animation inside of an `aria-hidden`, absolutely positioned element with a white background above the real placeholder (which will always contain the full text).
